src/Microsoft.Extensions.FileProviders.Physical/PhysicalFileProvider.cs

```diff
@@ -110,8 +125,17 @@ private bool IsUnderneathRoot(string fullPath)
         /// Locate a file at the given path by directly mapping path segments to physical directories.
         /// </summary>
         /// <param name="subpath">A path under the root directory</param>
-        /// <returns>The file information. Caller must check Exists property. </returns>
+        /// <returns>The file information. Caller must check <see cref="IFileInfo.Exists"/> property. </returns>
         public IFileInfo GetFileInfo(string subpath)
+            => GetFileInfo(subpath, _excludedFiles);
+
+        /// <summary>
+        /// Locate a file at the given path by directly mapping path segments to physical directories.
+        /// </summary>
+        /// <param name="subpath">A path under the root directory</param>
+        /// <param name="excludedFiles">Overrides default file exclusion behavior.</param>
+        /// <returns>The file information. Caller must check <see cref="IFileInfo.Exists"/> property. </returns>
+        public IFileInfo GetFileInfo(string subpath, ExcludedFiles excludedFiles)
```

Comment:
This feels weird - why have the options at two different levels? It doesn't seem intuitive, for instance, if I specified `ExcludedFiles.Hidden` in the ctor, and `ExcludedFiles.System` here, I'd sort of expect it to filter both, not just the latter.